res2df/compdat.py

```diff
@@ -950,7 +950,7 @@ def fill_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
     Arguments:
         parser: parser to fill with arguments
     """
-    parser.add_argument("DATAFILE", help="Name of reservoir DATA file.")
+    parser.add_argument("DATAFILE", help="Name of resdata .DATA file.")
```

Comment:
Just .DATA file can work or say `:term:input deck`

Reply:
Had a go at this:
https://github.com/equinor/ecl2df/pull/462/commits/da0b0a99f146f2d18694aab7013cb74222e4a41d